src/components/CowCard/CowCard.js

```diff
@@ -123,7 +123,7 @@ export const CowCard = (
     isCowOfferedForTradeByPeer && cowIdOfferedForTrade.length > 0
   )
 
-  const { isMounted } = useMountState()
+  const isMounted = useIsMounted()
```

Comment:
As of https://github.com/jeremyckahn/farmhand/commit/72d110b2155f1df3d7f631c27573997aa591da82, this is now the solution to the root cause of #527.